packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -74,7 +78,19 @@ const pollResult = async (runId: string) => {
 	poller
 		.setInterval(5000)
 		.setThreshold(100)
-		.setPollAction(async () => pollAction(runId));
+		.setPollAction(async () => pollAction(runId))
+		.setProgressAction((data: Simulation) => {
+			if (runId === props.node.state.inProgressOptimizeId && data.updates.length > 0) {
+				const checkpoint = _.first(data.updates);
+				if (checkpoint) {
+					const state = _.cloneDeep(props.node.state);
+					state.currentProgress = +((100 * checkpoint.data.progress) / checkpoint.data.totalPossibleIterations).toFixed(
+						2
+					);
+					emit('update-state', state);
+				}
+			}
```

Comment:
So I don't have a good notion of the frequency of updates in optimize, now that we have some sense of how fast the numbers changes - you may want to take the opportunity to tune the poller parameters:

```
		.setInterval(5000)
		.setThreshold(100)
```

E.g. if it is the case that we see changes every 10 seconds, maybe we don't need to poll as often.